curator/static/js/study-editor.js

```diff
@@ -6436,11 +6421,21 @@ function getNodeConflictDescription(tree, node) {
                 // EXAMPLE:  https://tree.opentreeoflife.org/opentree/argus/ottol@770315/Homo-sapiens
                 if (isNaN(node.conflictDetails.witness)) {
                     // it's a synthetic-tree node ID (e.g. 'ott1234' or 'mrcaott123ott456')
-                    witnessURL = "/opentree/argus/synth@{NODE_ID}".replace('{NODE_ID}', node.conflictDetails.witness);
-                    /* N.B. Ideally, instead of 'synth' above we would use the current 
-                     * synth-version (e.g. 'opentree7.0'). But anything other than 
-                     * 'ottol' should show this node ID in the latest synthetic tree.
-                     */
+                   /* N.B. Ideally we'd include the current synth-version (e.g. '/opentree7.0@ott123'),
+                    * like so:
+
+                    witnessURL = "/opentree/argus/{SYNTH_VERSION}@{NODE_ID}"
+                        .replace('{SYNTH_VERSION}', referenceTreeVersion)
+                        .replace('{NODE_ID}', node.conflictDetails.witness);
+
+                    * But this is not yet provided by the conflict service. Perhaps we could capture
+                    * this as <tree>.conflictDetails['referenceTreeVersion']
+                    *
+                    * For now, omitting the version entirely (e.g. '/@ott123') will redirect to
+                    * the latest-version URL.
+                    */
```

Comment:
Not too hard, but I don't want to do this in a hurry and want to merge this PR this weekend. I'll make a separate issue for versions.